### PR TITLE
126 create categorie dropdown component

### LINKED ISSUE #126: create categorie dropdown component

### PR DESCRIPTION
**Pull request by LetsPoke**
-added a category dropdown component

closes #126

### CODE REVIEW

File: hms-app/src/components/CatDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+function CatDropdown() {
```
Comment (verzano) [inline hms-app/src/components/CatDropdown.tsx:3]: We should avoid abbreviated names, right now this sounds like it is a dropdown for cats :cat:

Comment (LetsPoke) [reply hms-app/src/components/CatDropdown.tsx]: a cat dropdown sound great for me but i see the problem 😅

File: hms-app/src/TestIdeaData.js [hunk 2]
```diff
@@ -1,39 +1,38 @@
...
-        reason: "my reason was...",
-        problem: "it solve this and this problem",
-        goal: "the goal for the hackweek is...",
-        skills: "usefull skills are:",
-        categorie: "testdata",
-        participants: "me and myself",
-        aws: "need aws yes/no"
-    },
-
+  {
```
[the remark this answers is not in the document]
Comment (LetsPoke) [reply hms-app/src/TestIdeaData.js:2]: makes sense for later use/edit so thanks for the hint

File: hms-app/src/test/TestIdeaData.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+export default [
```
Comment (verzano) [inline hms-app/src/test/TestIdeaData.js:1]: ah, but put `test` one level higher so `src` and `test` are at the same level :)

Comment (LetsPoke) [reply hms-app/src/test/TestIdeaData.js:1]: not sure if this would cause build errors in react?

Comment (LetsPoke) [reply hms-app/src/test/TestIdeaData.js:1]: image

i would prefer to have it in /src directory

**Review by jakepanter — APPROVED** [review]

**Pull request merged**